pkg/cleaner/aws/aws.go

```diff
@@ -370,7 +370,7 @@ func (a *Cleaner) disableMasterTerminationProtection(stackName string) error {
 		return microerror.Mask(err)
 	}
 
-	if len(o.Reservations) != 1 {
+	if len(o.Reservations) > 1 {
```

Comment:
With HA Masters in mind, why not just stopping here when there are 0? If there are one or more we just proceed in a loop below.

Reply:
Thanks that is better and more future proof.

Done. PTAL